src/lib/idb.ts

```diff
@@ -374,97 +273,85 @@ export default class IDBStorage {
     return blob.size || blob.byteLength || blob.length;
   } */
 
-  public get<T>(entryName: string): Promise<T> {
+  public get<T>(entryName: string[]): Promise<T[]>;
+  public get<T>(entryName: string): Promise<T>;
+  public get<T>(entryName: string | string[]): Promise<T> | Promise<T[]> {
     //return Promise.reject();
-    return this.openDatabase().then((db) => {
-      //this.log('get pre:', fileName);
-
-      try {
-        const transaction = db.transaction([this.storeName], 'readonly');
-        /* transaction.onabort = (e) => {
-          this.log.error('get transaction onabort?', e);
-        }; */
-        const objectStore = transaction.objectStore(this.storeName);
-        var request = objectStore.get(entryName);
-        
-        //this.log.log('IDB get:', fileName, request);
-      } catch(err) {
-        this.log.error('get error:', err, entryName, request, request.error);
-      }
 
-      return new Promise((resolve, reject) => {
-        const timeout = setTimeout(() => {
-          this.log.error('get request not finished!', entryName, request);
-          reject();
-        }, 3000);
-
-        request.onsuccess = function(event) {
-          const result = request.result;
-          if(result === undefined) {
-            reject('NO_ENTRY_FOUND');
-          } /* else if(typeof result === 'string' &&
-            result.substr(0, 5) === 'data:') {
-            resolve(dataUrlToBlob(result));
-          }  */else {
-            resolve(result);
-          }
+    if(!Array.isArray(entryName)) {
+      entryName = [].concat(entryName);
+    }
 
-          clearTimeout(timeout);
-        }
-  
-        request.onerror = () => {
-          clearTimeout(timeout);
-          reject();
-        };
-      });
-    });
+    return this.getObjectStore<T>('readonly', (objectStore) => {
+      return (entryName as string[]).map((entryName) => objectStore.get(entryName));
+    }, 'get: ' + entryName.join(', '));
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/MikeOwino/tweb/pullRequest?prid=7329411)
```suggestion
    }, "get: " + entryName.join(', '));
```